doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,227 @@
 .. _readme-buildingdocs:
 
+.. _Chapel Developers: https://chapel.discourse.group/c/developers
+
 =============================
 Building Chapel Documentation
 =============================
+The live Chapel web documentation lives at https://chapel-lang.org/docs/.
+All of the source for the documentation is in the chapel source repository, and
+this note talks about how those are organized and built into the html documentation
+seen on the website.
 
-The easiest way to build the Chapel web documentation is to use the top level
-make target::
-
-   [g]make docs
+The easiest way to build the Chapel web documentation is to run the command
+``make docs`` in the top level directory of your ``chapel`` Github clone.
 
 This will build the documentation for the standard and internal modules using
 chpldoc, and symlink all of the ``$CHPL_HOME/doc/`` documents over to
 a sphinx project that lives in ``$CHPL_HOME/doc/``. Once built,
 the documentation can be viewed at ``$CHPL_HOME/doc/html/index.html``.
+
+The html documentation is also generated by the Github CI (continuous integration)
+process.  You can go to your pull request in Github, click on "Checks",
+then "Ci" (on the left), and then scroll down to see the "documentation"
+(at the very bottom).  It enables you to download a zip file of the built
+documentation.
```

Comment:
Maybe add: "This is particularly useful when reviewing someone else's PR and wanting to see what the resulting documentation looks like (where the alternative is to check out their branch and build it yourself)."

Reply:
added